.github/workflows/ci.yml

```diff
@@ -40,9 +40,18 @@ jobs:
         isort --check --diff .
     - name: Install Additional Dependencies for Type-Checking/Tests
       run: poetry install -E ci --no-interaction
-    - name: Check with mypy
+
+    - name: Setup node.js (for pyright)
+      uses: actions/setup-node@v1
+      with:
+        node-version: "12"
```

Comment:
how did you decide on 12? and not, for example, 14?

Reply:
I just copied this block from palbolts (where it's commented out)